Lib/shutil.py

```diff
@@ -1127,6 +1127,7 @@ def make_archive(base_name, format, root_dir=None, base_dir=None, verbose=0,
             if not dry_run:
                 os.chdir(root_dir)
 
+    base_name = os.fspath(base_name)
```

Comment:
Please do it only if `root_dir is not None`, and only if `support_root_dir` is true (because `os.path.abspath()` already does the work).

It is an undocumented behavior and we do not want to introduce a new official feature in a bugfix release.

Reply:
Thank you, amended.